x-pack/test/api_integration/deployment_agnostic/apis/observability/alerting/custom_threshold/helpers/syntrace.ts

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Client } from '@elastic/elasticsearch';
+import {
+  ApmSynthtraceEsClient,
+  ApmSynthtraceKibanaClient,
+  createLogger,
+  LogLevel,
+} from '@kbn/apm-synthtrace';
+import { apm, timerange } from '@kbn/apm-synthtrace-client';
+
+export const getSyntraceClient = async ({
```

Comment:
For Logs Explorer Synthtrace client, we did something like this

https://github.com/elastic/kibana/blob/0d19367fdfad5526b5220dfdf18b4991fe6b3abd/x-pack/test/api_integration/deployment_agnostic/services/logs_synthtrace_es_client.ts#L11-L19

With this, this client is accessible to all other tests if required

Reply:
@crespocarlos already added synthtrace service in his [PR](https://github.com/elastic/kibana/pull/198257), I will refactor this later when that PR is merged.